scripts/busca.js

```diff
@@ -76,17 +76,24 @@ module.exports = function(robot) {
         res.reply(':thinking_face: I can\'t find any upcoming events for ' + name + '.');
         return;
       }
-
+      edates = [];
```

Comment:
This declares a global variable. We should limit to local function scope. Consider `var edates = [];` to limit to the local scope.